apps/fz_common/lib/fz_net.ex

```diff
@@ -48,6 +48,21 @@ defmodule FzCommon.FzNet do
     end
   end
 
+  def inet_to_ip_with_mask(%{address: address, netmask: netmask} = inet, network) do
+    if netmask != nil do
+      inet
+    else
+      "#{address |> Tuple.to_list() |> Enum.join(".")}/#{get_range!(network)}"
+    end
+  end
+
+  def get_range!(inet) when is_binary(inet) do
+    case String.split(inet, "/") do
+      [_, mask] -> mask
+      _ -> raise "inet given #{inet} doesn't have a range"
+    end
+  end
```

Comment:
Hmm does the [EctoNetwork.INET module](https://github.com/adam12/ecto_network/blob/v1.3.0/lib/ecto_network/inet.ex#L22) already have functions for handling this logic?